src/App.spec.tsx

```diff
@@ -354,16 +354,29 @@ describe('App', () => {
         setup({ hasLoggedInUser: true, hasSession: true })
       })
 
-      it(`renders the ${testLabel} page`, async () => {
-        render(<App />, { wrapper: wrapper([pathname]) })
+      if (testLabel === 'AccountSettings') {
+        it.skip(`renders the ${testLabel} page`, async () => {
```

Comment:
if not too annoying, we can throw in a quick comment
`mysteriously flaky test, skip for now, see https://github.com/codecov/engineering-team/issues/2270`